acceptance/lib/helper.rb

```diff
@@ -1,4 +1,6 @@
 require 'beaker/dsl/install_utils'
+require 'puppetserver/acceptance/compat_utils'
+require 'puppetserver/acceptance/gem_utils'
```

Comment:
+1 for putting the require's everywhere they're actually needed rather than always just in the `helper.rb`